app/views/admin/products_v3/_content.html.haml

```diff
@@ -1,5 +1,5 @@
 %turbo-frame#products-content{ target: "_top", refresh: "morph" }
-  .spinner-overlay{ "data-controller": "loading", "data-products-target": "loading", class: "hidden" }
```

Comment:
This CSS is handling loader display now:
https://github.com/openfoodfoundation/openfoodnetwork/blob/337eae43a66ef9254ca9f6fe4d9650a86eb76ae3/app/webpacker/css/admin_v3/components/spinner.scss#L10-L13

Seems like we don't need loading controller here anymore.